src/plugins/new-ui/public/webcomponents/depth.html

```diff
@@ -191,11 +208,11 @@
           },
           target: {
             type: Number,
-            value: 0
+            value: function(){return 0;}
           },
           waterType: {
             type: String,
-            value: 'salt'
+            value: function(){return 'Saltwater';}
```

Comment:
The words freshwater and saltwater take up too mush space. A simple fresh/salt should do.